cmd/crictl/config.go

```diff
@@ -45,3 +50,80 @@ func ReadConfig(filepath string) (*Config, error) {
 	}
 	return &config, err
 }
+
+func writeConfig(c *Config, filepath string) error {
+	data, err := yaml.Marshal(c)
+	if err != nil {
+		return err
+	}
+	return ioutil.WriteFile(filepath, data, 0644)
+}
+
+var configCommand = cli.Command{
+	Name:      "config",
+	Usage:     "Get and set crictl options",
+	ArgsUsage: "[<options>]",
+	Flags: []cli.Flag{
+		cli.StringFlag{
+			Name:  "get",
+			Usage: "get value: name",
+		},
+	},
+	Action: func(context *cli.Context) error {
+		configFile := context.GlobalString("config")
+		if _, err := os.Stat(configFile); err != nil {
+			// can't continue w/o config file
+			logrus.Fatalf("Falied to load config file: %v", err)
```

Comment:
Can we create an empty config here instead?

Reply:
fixed